cms/core/checks.py

```diff
@@ -29,3 +33,22 @@ def check_wagtail_settings(*args: Any, **kwargs: Any) -> Iterator[CheckMessage]:
                 hint=f"Ensure generic setting extends {BaseGenericSetting!r}",
                 obj=model,
             )
+
+
+@register(Tags.models)
+def check_wagtail_pages(*args: Any, **kwargs: Any) -> Iterator[CheckMessage]:
+    for model in apps.get_models():
+        if not issubclass(model, Page):
+            continue
```

Comment:
suggestion: use `get_page_models` https://github.com/wagtail/wagtail/blob/988a48a75adc5a8ac0b68da9e65062349bad15b2/wagtail/models/pages.py#L107

Reply:
Changed in [9afa0fb](https://github.com/ONSdigital/dis-wagtail/pull/192/commits/9afa0fb22387597083060e80d701f986151c2a6a)